regexes.ipynb

```diff
@@ -95,6 +95,319 @@
+    "        elif word == \"warriors\":\n",
+    "            n_warriors += 1\n",
+    "        elif word == \"may\":\n",
+    "            n_may += 1\n",
+    "        elif word == \"your\":\n",
+    "            n_your += 1\n",
+    "        elif word == \"vows\":\n",
+    "            n_vows += 1\n",
+    "        elif word == \"be\":\n",
+    "            n_be += 1\n",
+    "        elif word == \"crown’d\":\n",
+    "            n_crown_d += 1\n",
+    "        elif word == \"troy’s\":\n",
+    "            n_troy_s += 1\n",
+    "        elif word == \"proud\":\n",
+    "            n_proud += 1\n",
+    "        elif word == \"walls\":\n",
+    "            n_walls += 1\n",
+    "        elif word == \"lie\":\n",
+    "            n_lie += 1\n",
+    "        elif word == \"level\":\n",
+    "            n_level += 1\n",
+    "        elif word == \"with\":\n",
+    "            n_with += 1\n",
+    "        elif word == \"the\":\n",
+    "            n_the += 1\n",
+    "        elif word == \"ground\":\n",
+    "            n_ground += 1\n",
+    "        elif word == \"jove\":\n",
+    "            n_jove += 1\n",
+    "        elif word == \"restore\":\n",
+    "            n_restore += 1\n",
+    "        elif word == \"you\":\n",
+    "            n_you += 1\n",
+    "        elif word == \"when\":\n",
+    "            n_when += 1\n",
+    "        elif word == \"toils\":\n",
+    "            n_toils += 1\n",
+    "        elif word == \"are\":\n",
+    "            n_are += 1\n",
+    "        elif word == \"o’er\":\n",
+    "            n_o_er += 1\n",
+    "        elif word == \"safe\":\n",
+    "            n_safe += 1\n",
+    "        elif word == \"to\":\n",
+    "            n_to += 1\n",
+    "        elif word == \"pleasures\":\n",
+    "            n_pleasures += 1\n",
+    "        elif word == \"of\":\n",
+    "            n_of += 1\n",
+    "        elif word == \"native\":\n",
+    "            n_native += 1\n",
+    "        elif word == \"shore\":\n",
+    "            n_shore += 1\n",
+    "        elif word == \"but\":\n",
+    "            n_but += 1\n",
+    "        elif word == \"oh\":\n",
+    "            n_oh += 1\n",
+    "        elif word == \"relieve\":\n",
+    "            n_relieve += 1\n",
+    "        elif word == \"a\":\n",
+    "            n_a += 1\n",
+    "        elif word == \"wretched\":\n",
+    "            n_wretched += 1\n",
+    "        elif word == \"parent’s\":\n",
+    "            n_parent_s += 1\n",
+    "        elif word == \"pain\":\n",
+    "            n_pain += 1\n",
+    "        elif word == \"give\":\n",
+    "            n_give += 1\n",
+    "        elif word == \"chryseïs\":\n",
+    "            n_chryseis += 1\n",
+    "        elif word == \"these\":\n",
+    "            n_these += 1\n",
+    "        elif word == \"arms\":\n",
+    "            n_arms += 1\n",
+    "        elif word == \"again\":\n",
+    "            n_again += 1\n",
```

Comment:
Great job -- this is definitely a valid solution to the problem. Take a look at some of the other pull requests to see a slightly more succinct way of handling this. Alternatively, how might you do this now that you know about dictionaries?